test/system.jl

```diff
@@ -14,7 +14,7 @@
         ids=gen_ids, datetimes=datetimes
     )
 
-    branch_names = string.([1, 2, 3])
+    branch_names = string.([1, 2, 3, 4])
```

Comment:
this is defined quite far away from where it is used... can we just remove it?

then we can create dictionaries like e.g.
```diff
- Dictionary(branch_names, branches)
+ index(br -> br.name, branches)
```

alternatively can we move it to where it is used? i think it'd make tests clearer and easier to maintain